pyron-core/src/main/scala/com/cloudentity/pyron/openapi/OpenApiUtils.scala

```diff
@@ -51,13 +48,13 @@ trait OpenApiConverterUtils {
     Option(path.getOperationMap.get(method))
 
   def buildPaths(pathsMap: Map[String, OpenApiOperations]): Map[String, Path] = {
-    pathsMap.mapValues(buildPath(_))
+    pathsMap.mapValues(buildPath)
   }
 
   def pathMatches(testPath: String, regexPath: String): Boolean = {
-    val normalizedTargetServicePath = testPath.replaceAll("[{}]", "")
-    val pathMatching = PathMatching.build(PathPrefix(""), PathPattern(regexPath))
-    PathMatcher.makeMatch(normalizedTargetServicePath, pathMatching).isDefined
+    regexPath
```

Comment:
Previously it would match, when it shouldn't, because if someone puts free-floating '{' or '}' they would be just ignored as if they are not there. This change, only removes [a-zA-Z_0-9]* surrounded by {} which is actual, valid param reference. The rest should and now must match, if it still contains '{' and our target path doesn't contain '{' the match should fail.